### PR TITLE
Blaze mod JSON cleanup

### PR DESCRIPTION
**Pull request by Night-Pryanik**
Extensively used `copy-from`. Also fixed several id dupes and decapitalized some names.

### CODE REVIEW

File: data/mods/blazemod/blaze_autoweapons_parts.json [hunk 1]
```diff
@@ -1,428 +1,337 @@
 [
+  {
+    "abstract": "turret_blaze",
```
Comment (illi-kun) [inline data/mods/blazemod/blaze_autoweapons_parts.json]: Adding this allows to reduce the file even more, isn't it?
```
     "color": "dark_gray",
     "broken_color": "dark_gray",
```

Comment (Night-Pryanik) [reply data/mods/blazemod/blaze_autoweapons_parts.json]: There are different colors for some turrets, for example `abolter` has `light_gray` colors.

Comment (Night-Pryanik) [reply data/mods/blazemod/blaze_autoweapons_parts.json]: Okay, done.

Comment (illi-kun) [reply data/mods/blazemod/blaze_autoweapons_parts.json]: Yeaah, diff +2 −282 looks great  :scissors:  :thumbsup:

**Pull request merged**